### PR TITLE
Limit bugfix and style fixes for collections

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| Enhancement? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | https://github.com/massiveart/husky/pull/721
| License | MIT
| Documentation PR | -

#### What's in this PR?

This PR fixes the issue that only 10 collections get returned from the API. This is achieved by disabling the limit and offset in the CollectionController.

*Style fixes*:
- A husky version is included which improves the cropping of the tile in the collection folders view  
- The "Move folder" button is now a dropdown item of "Edit folder"
- Icons were added to the breadcrumb
- The offset for the sticky-toolbar was hardcoded, which lead to jumping effects. This is fixed.
- The safari browser had a nasty jumping glitch when adding masonry tiles to the view with the infinite scroll effect. Safari showed the tiles before they where correctly positioned. Moving the task of showing the items to the end of the execution stack (with a 0 delay) fixed the issue.

### CODE REVIEW

**Comment by marcelmoosbrugger** [comment]
@danrot This PR can be reviewed.

File: src/Sulu/Bundle/AdminBundle/Resources/public/js/components/header/main.js [hunk 1]
```diff
@@ -112,7 +112,11 @@ define([], function() {
                 '   </div>',
                 '</div>'
             ].join(''),
-            breadcrumb: '<span class="breadcrumb"><%= title %><span class="fa-chevron-right icon"></span></span>'
+            breadcrumb: ['<span class="breadcrumb">',
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/public/js/components/header/main.js]: new line after `[`

File: src/Sulu/Bundle/AdminBundle/Resources/public/js/components/header/main.js [hunk 2]
```diff
@@ -112,7 +112,11 @@ define([], function() {
...
+                         '<% if (!!icon) { %><span class="<%= icon %> icon"></span><% } %>',
+                         '<%= title %>',
+                         '    <span class="fa-chevron-right separator"></span>',
+                         '</span>'].join('')
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/public/js/components/header/main.js]: new line before `]`

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/collection-select-overlay/main.js [hunk 3]
```diff
@@ -161,7 +161,7 @@ define(function() {
                         'collections': [
                             {
                                 'id': 'root',
-                                'title': this.sandbox.translate('navigation.media.collections'),
+                                'title': this.sandbox.translate('sulu.media.all'),
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/collection-select-overlay/main.js]: maybe `all-collections`?

**Comment by marcelmoosbrugger** [comment]
@danrot The issues mentioned in your comments are fixed now :-)

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js [hunk 4]
```diff
@@ -224,21 +222,31 @@ define([
...
             if (SecurityChecker.hasPermission(this.data, 'add')) {
                 this.startDropzone();
             }
 
-            this.startChildrenTiles();
-            this.startDatagrid();
+            deferreds.push(this.startChildrenTiles());
+            deferreds.push(this.startDatagrid());
+
+            $.when.apply($, deferreds).then(function() {
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js]: Why not simply `$.when(deferreds)`?

**Comment by danrot** [comment]
Is it supposed to look like that?
image

**Pull request merged**